### ISSUE TITLE
Are monetary donations possible?

### ISSUE THREAD

**Issue by Cimiteriophiliac** [veusz/veusz#604]
Veusz is one of the rare pieces of software that I immediately fell in love with. A lovely, straightforward interface that ACTUALLY offers all the options I need. So I looked for some kind of paypal link on various locations to give back a little bit, but couldn't find one. Does anything like this exist?

**Comment by kaugummiheld** [comment]
I second this wish. I'm using Veusz for years and always happy to see new functions or bug fixes. 
Unfortunately I won't be able to contribute code, but I'd like to give something back.

**Comment by IgnacioMonge** [comment]
> Veusz is one of the rare pieces of software that I immediately fell in love with. A lovely, straightforward interface that ACTUALLY offers all the options I need. So I looked for some kind of paypal link on various locations to give back a little bit, but couldn't find one. Does anything like this exist?

I totally agree. Veusz is one of the most useful and easiest software for scientific plotting. I would also love to contribute with my money to the development.
